src/Assets/LightSkyDefense/Scripts/Credit.cs

```diff
@@ -17,7 +23,11 @@ private void OnHandHoverBegin(Hand hand)
     {
         var player = Player.instance.gameObject;
         var creditOwner = player.GetComponent<CreditOwner>();
-        creditOwner?.AddCredits(value);
+        if (creditOwner == null) return;
+        creditOwner.AddCredits(Value);
+        source.PlayOneShot(pickupSound);
         Destroy(gameObject);
```

Comment:
Also around member or static method invokation

```suggestion

        Destroy(gameObject);
```